.github/workflows/ci.yaml

```
@@ -101,8 +101,6 @@ jobs:
             git push origin HEAD:main -f
 
         
-        
-          
-
+###########################################################################################################################################################################
 
-        
\ No newline at end of file
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)